samples/wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+# This file is part of BOINC.
+# http:#boinc.berkeley.edu
+# Copyright (C) 2014 University of California
+#
+# BOINC is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# BOINC is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with BOINC.  If not, see <http:#www.gnu.org/licenses/>.
+#
+# Contributor: Anna Vyalkova (cyber@sysrq.in)
+
+cmake_minimum_required(VERSION 3.12)
+
+project(boinc-wrapper
+    VERSION 26019
```

Comment:
We have a script that automatically set version to wrapper: https://github.com/BOINC/boinc/blob/master/set-wrapper-version.py
Please modify it to include this file